subsys/stats/Kconfig

```diff
@@ -0,0 +1,22 @@
+# Kconfig - mcumgr configuration options
+
+#
+# Copyright Runtime.io 2018. All rights reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+config STATS
+	bool
+	prompt "Statistics support"
+	default n
+	help
+	  Support for per-module statistics.
```

Comment:
This needs more documentation.

It needs to be clear to an application developer that discovers this subsystem through a Kconfig interface what happens if he enables or disables this.

Reply:
Thanks, I have added some more detail.